sdcm/nemesis.py

```diff
@@ -5438,6 +5420,67 @@ def drop_keyspace(node):
                 LOGGER.debug("Query result %s", result)
                 assert not result, f"New rows were added from banned node, {result}"
 
+    @target_all_nodes
+    def disrupt_kill_mv_building_coordinator(self):
```

Comment:
please add docstring describing the nemesis

Reply:
added